src/internal/plugins/controllers/drawers.ts

```diff
@@ -17,15 +17,14 @@ export interface DrawerStateChangeParams {
 
 export interface DrawerConfig {
   id: string;
-  type?: 'local' | 'global' | 'global-ai';
+  type?: 'local' | 'global';
```

Comment:
All of the diff in `src/internal/plugins` is to return the code back to its mainline state.

Can be confirmed in this diff: https://github.com/cloudscape-design/components/pull/3795

<img width="278" height="170" alt="Image" src="https://github.com/user-attachments/assets/a63d81d2-fb8a-4fae-90be-243520839819" />